packages/template-retail-react-app/jest-setup.js

```diff
@@ -49,7 +49,7 @@ export const setupMockServer = () => {
             res(
                 ctx.delay(0),
                 ctx.json({
-                    customer_id: 'test',
+                    customer_id: 'customerid',
```

Comment:
Need to keep this consistent with the mock data (mockRegisteredCustomer has customerId as 'customerid')